src/Service.Host/client/src/components/salesPackages/SalesPackages.js

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Loading, CreateButton } from '@linn-it/linn-form-components-library';
+import { Table, TableHead, TableBody, TableRow, TableCell } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import EditIcon from '@material-ui/icons/Edit';
+import Page from '../../containers/Page';
+
+function SalesPackages({ items, loading }) {
+    const [rowOpen, setRowOpen] = useState();
+
+    const handleRowOnClick = salesPackageId =>
+        rowOpen === salesPackageId ? setRowOpen(null) : setRowOpen(salesPackageId);
+
+    const cursor = {
+        cursor: 'pointer'
+    };
+
+    const identifySelfLink = row => {
```

Comment:
We have a getSelfHref method in the common folder which you could use for this

Reply:
👍